examples/lump-mpi.py

```diff
@@ -47,7 +47,8 @@
 from mirgecom.mpi import mpi_entry_point
 
 from mirgecom.integrators import rk4_step
-from mirgecom.steppers import advance_state
+from leap.rk import RK4MethodBuilder
```

Comment:
If you add this here, you make `leap` a hard dependency (which means it would need to be added throughout the installation infrastructure). I think it should suffice to import it only if it's actually used.

Additional thoughts;
- Grudge did have a hard dependency on leap, but: https://github.com/inducer/grudge/pull/69
- You'll need to add dagrt/leap to `requirements.txt` (like [here](https://github.com/inducer/grudge/blob/d090d7e3ad4fc63f67397257c5257c0c049cf4b0/requirements.txt#L8-L9)) to ensure they get picked up.